### DISCUSSION TITLE
//Question// Claude 3.5 Sonnet  and Gemini Flash support?

### DISCUSSION THREAD

**Discussion by pertevniyal** [danny-avila/LibreChat#3168] [Q&A] [answered]
Hello! I was just wondering if Claude 3.5 Sonnet  and/or Gemini Flash support will be added with 0.7.4?
Also, the option for Gemini Pro doesn't seem to indicate whether it is Pro 1.5 or Pro 1.0, I was wondering if someone could clarify that.

Thanks for your work and have a good day!

**Comment by danny-avila** [comment, marked as answer]
They're already supported, you just need to add them to your models list:
```.env
# .env file
ANTHROPIC_MODELS=claude-3-5-sonnet-20240620 # rest omitted
GOOGLE_MODELS=gemini-1.5-flash-latest # rest omitted
```

In general, new model updates don't require an update to the app, as long as you can configure your list. Updates to the app based on a new model release are more to update pricing/context limits.

Also here's a preview of an unrelated feature (multi-response streaming https://github.com/danny-avila/LibreChat/issues/3028).

I'm just using it showcase a response from both 3.5-sonnet and gemini-flash
![image](https://github.com/danny-avila/LibreChat/assets/110412045/346f0bcc-1d3b-47c1-8d36-4f5094bfe0ee)

**Comment by pertevniyal** [comment]
Huge hanks for the quick reply! I tried to do as instructed and edited the .env file and the line for 3.5 sonnet, although I tried everything I couldn't get the GUI to show the option for it. So I updated LibreChat as per the Local Installation in Documentation and now I have 3.5 sonnet showing in the GUI, but Gemini Flash is missing. After the update my .env file however doesn't show 3.5 sonnet but it does show Gemini Flash, which is the exact opposite of the GUI. I couldn't figure out what I'm missing. Thanks again for your help!

**Reply by danny-avila** [reply]
Are you using docker desktop? that would help to inspect your current environment variables, as well as restarting your container, as that needs to be done to take effect changes to the .env file
